content/post/057-ipfs-camp-course-videos.md

```diff
@@ -0,0 +1,80 @@
+title: IPFS Camp course videos released 🍿
+author: Teri Chadbourne
+---
+
+If you've been following the updates from IPFS Camp here in the blog, you've likely already explored the videos from our [lightning talks](https://github.com/ipfs/camp#%EF%B8%8F-lightning-talks), [deep dives](https://github.com/ipfs/camp#-deep-dives), and [poster projects](https://github.com/ipfs/camp#-poster-projects). **Today we're excited to announce the release of the videos from our [core and elective courses](https://github.com/ipfs/camp#-core--elective-courses)**!
+
+## Core Courses
+Core courses at IPFS Camp served as the starting point to get all of us on the same page with a shared understanding of some the most essential concepts underlying IPFS. Elective courses covered even more ground, branching out into bleeding-edge DWeb topics, exciting use cases, and community endeavors. The format for these 75-minute courses varied, so you'll find app-building tutorials and  interactive group discussions alongside cleverly diagrammed deep-dives and interpretive skits. There's something for everyone!
+
+**Check out the [complete course listings](https://github.com/ipfs/camp#-core--elective-courses) in the IPFS Camp repo for links to videos, slide decks, and other workshop materials for each session.**
+
+Whether you're new to IPFS or looking for a deeper understanding of core concepts, we highly recommend these two core courses as your starting point. Both are easily consumable in video form -- no coding required -- and offer plenty of "aha" moments:
+
+### Understanding How IPFS Deals with Files
+This core course covers the basics of how IPFS deals with files, including key concepts like immutability, content addressing, hashing, the anatomy of CIDs, what the heck a Merkle DAG is, and how chunk size affects file imports. It also covers the joys and pitfalls of the Mutable File System (MFS), the layer of abstraction in IPFS that lets you work with immutable files and directories as if you were using a tradiitional name-based file system. As with all great talks, this one is enhanced by images of adorable poodles. 🐩 (12 / 10, would recommend.)
+
+This workshop introduced a few awesome visualization and learning tools that you can try out at home:
+
+- Explore the [ProtoSchool tutorial on the Mutable File System (MFS)](https://proto.school/#/mutable-file-system) for coding challenges designed to familiarize you with its API
+- Paste a Content Identifier (CID) into the [CID Inspector](https://cid.ipfs.io/) to break down its anatomy
+- Drag a file into the [DAG Builder](https://dag.ipfs.io/) and adjust chunk size, DAG layouts, etc. to see how it affects the Merkle DAG
+
+Watch the recording below or check out the [slides](https://github.com/ipfs/camp/blob/master/CORE_AND_ELECTIVE_COURSES/CORE_COURSE_A/IPFS_Camp_Core_Course_A_Slides.pdf)!
+
+[![Video: Understanding How IPFS Deals with Files](/img/057-ipfs-camp-course-videos/core-a-thumbnail.png)](https://youtu.be/dJ7TkB4UpZ8)
```

Comment:
This image may need an update to the new one: https://drive.google.com/open?id=1sYpsu-K6DSSB81EICTlKXz0lQI_7pHBf

Reply:
done, thanks for the reminder!